src/components/Header.jsx

```diff
@@ -1,14 +1,16 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+import { CgProfile } from 'react-icons/cg';
 
 const Header = () => (
   <div>
     <nav>
       <h1>Bookstore CMS</h1>
-      <ul>
+      <ul className="nav-link">
         <li><Link to="/" style={{ textDecoration: 'none' }}>BOOKS</Link></li>
         <li><Link to="/categories" style={{ textDecoration: 'none' }}>CATEGORIES</Link></li>
       </ul>
+      <CgProfile className="book-avatar" />
     </nav>
   </div>
 );
```

Comment:
- [ ] Kindly update the styling here to match the design closer. Take a closer look on the colors, font of the elements
Yours: ![image](https://user-images.githubusercontent.com/62705947/191765440-8358d0fa-a8bd-43fc-951e-08dcb23eac30.png)
![image](https://user-images.githubusercontent.com/62705947/191765515-60b09472-7a9b-44f9-a6b2-7655540ff8ea.png)